xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -733,6 +734,18 @@ bool CVideoPlayerVideo::ProcessDecoderOutput(double &frametime, double &pts)
     if (m_speed != 0)
       pts += m_picture.iDuration * m_speed / abs(m_speed);
 
+    // ATSC A53 Closed Captions
+    if (m_picture.HasA53SideData() &&
+        CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+            CSettings::SETTING_SUBTITLES_PARSECAPTIONS))
+    {
+      auto a53SideData = m_picture.ConsumeA53SideData();
```

Comment:
I would say that this usage of auto isn't that great as the type isn't know from the method.
```suggestion
      AVBufferRef* a53SideData = m_picture.ConsumeA53SideData();
```